torch/fx/experimental/proxy_tensor.py

```diff
@@ -81,6 +80,8 @@ def set_proxy_slot(obj, tracer, proxy):
         # on a tensor, and it affects the metadata on the proxy.
         tracer.tensor_tracker[obj] = proxy
     else:
+        # Avoid importing sympy at a module level
+        from .symbolic_shapes import SymNode
```

Comment:
Why not move this PR higher in the stack to where `SymNode` can be imported at the top level?

Reply:
Yep, not sure why I didn't do that. In any case, I make it global in the next PR.